README.md

```diff
@@ -1 +1,19 @@
-# codespace
\ No newline at end of file
+# Webaverse Codespace
+Run your own Webaverse on Codespaces
+
+## Quick Start
+
+ - [ ] Open a terminal and do the following:
+ 
+```sh
+# You will answer some cert questions
```

Comment:
Can we skip the questions with flags?

Reply:
Also can we automate the script entirely so the user does not need to run it? Like as part of the install?

Reply:
yeah I had that initially in an earlier commit but I wasn't sure if we wanted to skip the questions on the certs. I'll make these changes.